src/renderer/components/editable-list/editable-list.scss

```diff
@@ -23,7 +23,7 @@
   .el-contents {
     display: flex;
     flex-direction: column;
-    margin-top: $padding * 2;
+    margin: $padding 0px;
```

Comment:
Any explanation why do we want to change EditableList styles?

Reply:
The `*/details.tsx` files use `<EditableList>` so these changes we to make it compatible with several of them in a column.